README.md

```diff
@@ -1,2 +1,136 @@
+-- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> In this project, you will set up a "Hello Microverse" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. 
+
+**Hello-Microverse** is a the first project built by the developer who is in the early stages of learning how to code. It desmostrate how to master all of the tools and best practices and how to code remotely to your github repository.
+
+Furthermore, you will learn how to use linters to fix errors in your code and make sure that your code is clean.
+
+## 🛠 Built With  <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <summary>Javascript runtime environment</summary>
+  <ul>
+    <li><a href="https://nodejs.org/en/">Node JS</a></li>
+  </ul>
+
+  <summary>Version control</summary>
+  <ul>
+    <li><a href="github.com">Git Hub</a></li>
+  </ul>
+</details
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Creating your first "Hello Microverse" project
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- **Visual Studio Code as editor**
+- **Git bash**
+- **github account**
+- **Markdown Preview Enhanced(optional)**
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd Hello-Hicroverse
+  git clone https://github.com/omer-creator/Hello-Microverse.git
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Muhammad Omer Afzal
+- GitHub: https:github.com/omer-creator
+- Twitter: https://twitter.com/Micronaut_Omer
+- LinkedIn: https://www.linkedin.com/in/muhammad-omer-afzal-67ab17250/
+## 🤝 Contributing <a name="contributing"></a>
+
+<!-- CONTRIBUTING -->
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please keep on coding. NEVER GIVE UP! We are here to collaborate.
+
+You can send a simple "thank you" message to the comment section.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="Microverse Inc."></a>
+
+I would like to thank Microverse team for guiding us through this project.
+
+I would also like to thank Joy and Fatuma for the good collaboration that we had in successfully completing this project.
+
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **[Question_1]**
+Where can we get the set up for node JS installation?
+
+  - [Answer_1]
+Please click on the link https://nodejs.org/en/download/  to get the  setup.
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
This link is broken because you don't have MIT file in your repo. Kindly set up your MIT license by using the [template ](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository)available on GitHub. ☺️ 📗